rbac/management/auditLogs/model.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Model for audit logging."""
+from django.db import models
+
+
+class AuditLogModel(models.Model):
```

Comment:
1. I believe that we have naming convention without model word. I would prefer `AuditLog`.
2. if model is using Tenancy (corresponding  table has tenant_id column) model inherits from `TenantAwareModel` for example like in role model `class Role(TenantAwareModel):`